src/components/home/modals/Send.vue

```diff
@@ -587,6 +587,14 @@ export default {
                 return void 0
             }
             let isValid = false
+            let alias = JSON.parse(window.localStorage.getItem(this.defaultAddress)).alias
+            for (let key in alias) {
+                if (recipient === alias[key]) {
+                    this.changeAliasToAddress(key)
```

Comment:
This design is not good. For example, if user have two friend, "John" and "Johny". He/She can not  input "Johny"'s address, because when he/she inputs 4 letters, it matched  "John" . So I suggest just check alias here, and change to "Alias(Address)" display in next confirm modal.

Reply:
Btw, the alias input match should change to case insensitive.